racer.el

```diff
@@ -104,13 +105,13 @@ If nil, we will query $CARGO_HOME at runtime."
   "Call racer command COMMAND with args ARGS."
   (let ((rust-src-path (or racer-rust-src-path (getenv "RUST_SRC_PATH")))
         (cargo-home (or racer-cargo-home (getenv "CARGO_HOME"))))
-    (when (null rust-src-path)
-      (user-error "You need to set `racer-rust-src-path' or `RUST_SRC_PATH'"))
     (let ((default-directory (or (racer--cargo-project-root) default-directory))
-          (process-environment (append (list
-                                        (format "RUST_SRC_PATH=%s" rust-src-path)
-                                        (format "CARGO_HOME=%s" cargo-home))
-                                       process-environment)))
+          (process-environment
+           (append
```

Comment:
as mentioned in #54, this is the first time I'm writing elisp, so I'd appreciate feedback if this is idiomatic.